site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,293 @@
+But, of course, you can bind these actions to other keys in Citra's `qt-config.ini`.
+The property for pausing is `Shortcuts\Main%20Window\Continue\Pause\KeySeq` and
+for speed toggling it is `Shortcuts\Main%20Window\Toggle%20Speed%20Limit\KeySeq`.
+
+<!-- Hidden as this isn't merged yet.
+## [Make hotkeys configurable via GUI](https://github.com/citra-emu/citra/pull/3786) by [Adityarup Laha](https://github.com/adityaruplaha)
+
+Although we have had hotkeys for a while now, not being able to configure them
+to our needs was a bummer. With this, you can now configure and customize
+hotkeys via the configuration window. However, we don't support binding hotkeys
+to controllers as of now. -->
+
+
+## [Add support for multiple game directories](https://github.com/citra-emu/citra/pull/3617) by [BreadFish64](https://github.com/BreadFish64)
+
+You can now configure Citra to search multiple directories for games and apps!
+This allow you to keep titles across different folders, and if you have any
+launchable titles installed into Citra's virtual SD card or NAND, they'll also
+show up in their own section so you can boot them from the game list.
+
+## [Add support for stereoscopic 3D](https://github.com/citra-emu/citra/pull/3632) by [N00byKing](https://github.com/N00byKing)
+
+If you are one of the lucky few with a 3D TV or monitor attached to your
+computer, stereoscopic 3D support has just been added to Citra! You can enable
+it by heading to <kbd><samp>Emulation</samp> &rarrow; <samp>Configure...</samp> &rarrow; <samp>Graphics</samp> &rarrow; <samp>Layout</samp></kbd>,
+ticking the <samp>Enable Stereoscopic 3D</samp> checkbox, and changing the
+screen layout to <samp>Side by Side</samp>.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/3d-sxs.png"
+    title="Although you can't see this in 3D, try crossing your eyes or using a mirror!" >}}
+
+If you want to take a look at the results, I've also prepared a rendering in
+the 3DS' native 3D image format. Download this file and drop it into your SD
+card, or navigate to this link on your 3DS to take a look:
+[](/images/entry/citra-progress-report-2018-q2/CTRA0001.MPO)
+
+## [Software Keyboard Implementation](https://github.com/citra-emu/citra/pull/3850) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+For the longest time, many games were unplayable or needed workarounds on Citra
+due to them needing the software keyboard applet. The reason being that whenever
+they would try to call it, Citra didn't implement it, and so would simply tell
+the game that the user entered "Citra" and tapped OK, without actually prompting.
+But now, thanks to [zhaowenlan1779](https://github.com/zhaowenlan1779) and [jroweboy](https://github.com/jroweboy),
+it now has fully functional software keyboard emulation!
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/swkbd.png"
+    title="Now the professor will finally stop calling you Citra!" >}}
```

Comment:
The image need to be provided.
(I heard @CaptV0rt3x wanted to use a gif to replace it though)